components/director/src/main.rs

```diff
@@ -253,6 +256,15 @@ fn dispatch(config: Config, matches: &clap::ArgMatches) -> Result<()> {
 }
 
 fn start(config: Config) -> Result<()> {
+    let mut ui = UI::default();
+    if !am_i_root() {
+        try!(ui.warn("Running the Habitat Supervisor root or administrator privileges. Please \
```

Comment:
should this say "Running the Habitat Supervisor **requires** root or admin ..."?